scielomanager/editorialmanager/models.py

```diff
@@ -35,7 +35,7 @@ class EditorialMember(models.Model):
     link_cv = models.URLField(_('Link CV'), null=True, blank=True)
     city = models.CharField(_('City'), max_length=256, null=True, blank=True)
     state = models.CharField(_('State'), max_length=256, null=True, blank=True)
-    country = CountryField(_('Country'), default='')
+    country = CountryField(_('Country'))
```

Comment:
> qual é a justificativa para o uso do param default='' no campo country

a justificativa de usar o default é trivial. Quando um campo não nullable é adicionado no model deve ter um atributo ``default`` no momento de fazer um schemamigration. Logo default para Charfield seria: ``default=""``.

> Este comentário "(por escola do plugin, e sem ele acredito que select normal faria a mesma coisa)"....., não é verdade! 

? 
um select normal ou seja, gerado a partir de um ``Charfield`` não nullable, aonde seu ``choices`` não contém uma opção 'blank', não vai gerar essa opção 'blank'  (``'------'``) entre os ``options`` do ``select`` gerado, logo, o plugin não vai adicionar essa opção  implicitamente, de fato é o que esta acontecendo. e de novo a opção 'blank' deve ser indicada explicitamente.

> Um select normal iria permitir a seleção de '------'

Um select normal só vai oferecer a opcão `` '------'`` se for definido explicitamente, o que não é o caso do ``CoutryField``.

Então resumindo toda esta conversa, não acho acertado remover um atributo do models, para resolver um problema de UI, e para isso existe uma atributo do ``CountryField`` como já indiquei.
Caso insista em manter a remoção do atributo pode proceder, eu não tenho mais comentários para agregar.